analysis.py

```diff
@@ -55,14 +56,26 @@ def setup_account(output: object, entry: dict, data_dir: str, plugin: object) ->
     return ac
 
 
-def check_account_with_spider(spac: deltachat.Account, account: deltachat.Account):
+def check_account_with_spider(spac: deltachat.Account, account: deltachat.Account, testfile: os.PathLike, timeout: int):
     """Send a message to spider and check if a reply arrives.
 
     :param spac: the deltachat.Account object of the spider.
     :param account: a deltachat.Account object to check.
-    """
+    :param testfile: path to test file, the default is 15 MB large
+    :param timeout: seconds until the script gives up
+   """
     chat = account.create_chat(spac)
-    chat.send_text("Begin: %f" % time.time())
+    # need to copy testfile to blobdir to avoid error
+    shutil.copy(testfile, account.get_blobdir())
+    newfilepath = os.path.join(account.get_blobdir(), os.path.basename(testfile))
+    message = chat.prepare_message_file(newfilepath, mime_type="application/octet-stream")
+    chat.send_prepared(message)
+    begin = time.time()
+    while message.is_out_delivered() is False and time.time() < begin + timeout:
+        time.sleep(0.5)
+    else:
+        # currently throws warning: The message Mr.X@testrun.org has no UID on the server to delete
+        account.delete_messages([message])
```

Comment:
Currently, the logs give us the following warning:

```
343.84 [events] DC_EVENT_WARNING data1=0 data2=src/job.rs:635: The message Mr.f872Ogn20p8.zDehgJF4ZDD@testrun.org has no UID on the server to delete
```

So these lines probably have no effect yet. Ideas why?

Reply:
smtp-outbound messages do not have a server-uid/imap folder. Only when core sees the (self-bcc) message will it update the server-uid/imap folder and then it can be deleted.  I am not sure if the deletion will ultimately happen.  Is it really needed to delete messages actually?

Reply:
No, it's more to clean up after oneself :) I think I just take out those lines on the main branch